source/docs/quick_start.md

```diff
@@ -1,61 +1,38 @@
+This will create a demo application. Feel free to look around to get familiar with the project's structure, but don't worry, we'll take a closer look at it later in one of our [dedicated guides](structure.html).
 
-Note: starting in Embark 3.1, the command `embark run` will automatically start a blockchain node if one isn't already running. You may, therefore, skip this step.
+### Running your app
 
-You can run a REAL ethereum node for development purposes:
-
-<pre><code class="shell">$ embark blockchain
-</code></pre>
-
-Alternatively, to use an ethereum rpc simulator simply run:
-
-<pre><code class="shell">$ embark simulator
-</code></pre>
-
-By default Embark blockchain will mine a minimum amount of ether and will only mine when new transactions come in.
+The fastest way to get your app running is to use Embark's `run` command. It takes care of a lot of things, such as spinning up an Ethereum and IPFS node, or keeping an eye on file changes to recompile your code. There's other ways to [run your app](running_apps.html), in case you need more control over different processes, but let's not get ahead of ourselves.
 
+<pre><code class="shell">$ embark run</code></pre>
 
-If you want, you can skip this step, as `embark run`, `build` and `upload` now all start a blockchain node in a separate process if there is not one already started using the same configurations. 
+Once executed, we'll notice that Embark opens up a dashboard view inside our terminal. This is the place where we monitor running processes such as compilation of our sources as well as deployments. As everything in Embark, the dashboard usage is configurable. If we prefer Embark to just output logs of whatever it's doing, this can be easily done by running the same command with the `--nodashboard` option.
 
-### Start Embark
-
-Then, in another command line:
-
-<pre><code class="shell">$ embark run
-</code></pre>
-
-This will open up the embark Dashboard
+Notice that the dashboard comes with sections for **Contracts**, **Environment**, **Status**, **Available Services**, **Logs** and **Console**. While most of them are self explanatory, we'll take a closer look at those in the [dashboard guide](dashboard.html).
 
 ![Dashboard](http://i.imgur.com/s4OQZpu.jpg)
 
-The dashboard will tell you the state of your contracts, the environment you are using, and what Embark is doing at the moment.
-
-**available services**
-
-Available Services will display the services available to your dapp in green. If a service is down, then it will be displayed in red.
+For now, let's focus on what has happened in the meantime. Embark has compiled and deployed the Smart Contracts that come with the demo application to a custom blockchain on your local machine. It has also compiled the web app that's part of the demo and deployed that to a local web server, which is listening on `http://localhost:8000`. In fact, Embark has probably already opened a browser window for you. 
```

Comment:
The dashboard screenshot in line 26 probably should be updated with a more "neutral" version. For example, the first time I saw the embark docs it looked like embark comes with a mountain landscape out of the box. 

Some people may not like this idea and might refuse to look further into the tool ( I know sounds silly, but let's just be sure)

Also, we should ensure that the font size in the screenshot is big enough so whatever is being shown is easy to read.

Reply:
This is done in https://github.com/embark-framework/embark-site/pull/82/commits/217b4554bb2e00822738d7375c9630b8572c6f70